content/posts/bayestestR_distribution.Rmd

```diff
@@ -52,24 +52,24 @@ x %>%
   geom_line()
 ```
 
-A few days before this update, we received a small email asking:
+Interestingly, a few days before this update, we received a small email asking:
 
 > Hi, I saw your blog post and wonder how you define a perfectly normal distribution.
 
-We responded, as honestly as we could:
+We responded:
 
 > that's actually a good question. I would say an empirical sample having characteristics as close as possible to a canonic Gaussian distribution.
 
 And just now, after an (ironic?) *"Thanks, most helpful!"*, [this blog post](https://xianblog.wordpress.com/2019/05/09/a-perfectly-normally-distributed-sample/) is published, which emphasize on the irrelevance of the `r` prefix in the function name. A great timing, as we wanted to share the update that we made! 
 
-Note that the `rnorm_perfect` name will be deprecated in the next version, and ultimately removed to avoid further confusion :)
+Thus, please note that the `rnorm_perfect` name will be deprecated in the next version, and ultimately removed to avoid further confusion :)
 
 
 ## Any other suggestions?
 
-Don't forget, the [easystats project](https://github.com/easystats) (that includes bayestestR) is very open to contributions and suggestions!
+Don't forget, the [easystats project](https://github.com/easystats) (that includes `bayestestR`) is very open to contributions and suggestions!
 
-Thus, if you ever plan on publishing personal correspondence with the response to a falsely naive question, please remember that you can always [make suggestions](https://github.com/easystats/bayestestR/issues) and contribute to the package, to *actually help* improving it, with the spirit of collaboration, and in the tradition of open science ;)
+Thus, if you ever plan on publishing personal correspondence of a response to a falsely naive question, please remember that you can always [make suggestions](https://github.com/easystats/bayestestR/issues) and contribute to the package, to *actually help* improving it, with the spirit of collaboration, and in the tradition of open science ;)
```

Comment:
I’m not convinced this last section is needed. I get the idea but it sounds a little vindictive. I think the previous line effectively opens the opportunity for contributions. Just my opinion.

Reply:
Yes this has been removed 😅